lib/netconf/upgrades.go

```diff
@@ -0,0 +1,82 @@
+package netconf
+
+import (
+	"errors"
+)
+
+const (
+	Virgil   = "virgil"
+	Ovid     = "v1.2.0"
+	Polybius = "polybius"
+
+	V121 = "v1.2.1"
```

Comment:
Is there a release name?  why V121 here? The others have names.

Reply:
@limengformal is right.